dll/win32/shell32/folders/CRecycleBin.cpp

```diff
@@ -897,7 +907,7 @@ TRASH_CanTrashFile(LPCWSTR wszPath)
 
     if (!GetVolumeInformationW(wszRootPathName, NULL, 0, &VolSerialNumber, &MaxComponentLength, &FileSystemFlags, NULL, 0))
     {
-        ERR("GetVolumeInformationW failed with %u\n", GetLastError());
+        ERR("GetVolumeInformationW failed with %u wszRootPathName[%s]\n", GetLastError(), debugstr_w(wszRootPathName));
```

Comment:
Suggestion: add comma after "%u", and don't use brackets around "%s" but use either an equal-sign or a colon.

Reply:
I like the brackets because it tells exactly how long the string is, but I will change it.